spec.bs

```diff
@@ -404,7 +404,7 @@ The <dfn for=Navigator method>runAdAuction(|config|)</dfn> method steps are:
 1. If |global|'s [=associated Document=] is not [=allowed to use=] the "[=run-ad-auction=]"
   [=policy-controlled feature=], then [=exception/throw=] a "{{NotAllowedError}}" {{DOMException}}.
 1. Let |auctionConfig| be the result of running [=validate and convert auction ad config=] with
-  |config| and [=validate and convert auction ad config/isTopLevel=] set to true.

```

Comment:
Heh, I actually prefer to keep the named arguments, and Infra endorses the idea: https://infra.spec.whatwg.org/#example-algorithm-optional-named-params. I don't insist either way.

Reply:
This is the comment in the spec review doc that suggested not giving isTopLevel a definition:
"Doesn't need isTopLevel to be its own definition (doing so is inconsistent with some of the other algorithms). It isn't necessarily wrong, but makes it harder to trace the variable references since you don't get the highlighting when you click"

Reply:
Ooh, the highlighting thing is an interesting point. I should file that as a bikeshed bug.